internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -149,10 +145,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				MarkdownDescription: "Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.\n\n**NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.",
 			},
 			"cluster_id": schema.StringAttribute{ // TODO: was generated as id
```

Comment:
can the TODO be addressed?

Reply:
TODOs will be handled as part of [the next Jira Issue](https://jira.mongodb.org/browse/CLOUDP-286481)